install/ingest_code_tables.py

```diff
@@ -1,34 +1,17 @@
 import os
 from sqlalchemy import create_engine, text
-from sqlalchemy_utils import database_exists, create_database, drop_database
 from sqlalchemy.orm import sessionmaker
-
 from opencdms.adapters import opencdmsdb
 
-
 # set connection details
 UID = os.environ["POSTGRES_USER"]
```

Comment:
At some stage we'll manage the database connection details in a central location and import a dictionary instead of duplicating these lines in multiple files - but I think it's fine for now.